analyzers/StylingAnalyzers.targets

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
   <ItemGroup>
+    <PackageReference Include="SonarAnalyzer.CSharp.Styling" Version="9.24.0.89429">
```

Comment:
Same [question](https://github.com/SonarSource/sonar-scanner-msbuild/pull/1912/files#r1582815610) as for sonar-scanner-msbuild.
Could this be updated when running the `set-version.ps1` script?